pibooth/pictures/__init__.py

```diff
@@ -26,14 +26,34 @@ def get_filename(name):
     return osp.join(osp.dirname(osp.abspath(__file__)), 'assets', name)
 
 
+def colorize_pil_image(pil_image, color, bg_color=None):
```

Comment:
This new feature is changing the behavior we have on the layouts, you should have put it in another PR...